src/server/api/routers/rooms.ts

```diff
@@ -33,6 +34,24 @@ export const roomsRouter = createTRPCRouter({
           id:randomUUID()
         }).returning();
       return res;
-    })
-    
+    }),
+    updateRooms: teacherProcedure
```

Comment:
Really good this should work perfectly. Just call this `update` though, that'll keep it inline with the other procedures on this router.

Reply:
Got it!